docs/developer/client.md

```diff
@@ -1,93 +1,85 @@
 When running a Software Delivery Machine (SDM) or lower-level Atomist
 API client, you interact with the Atomist development automation API
```

Comment:
"Atomist development automation API" should probably be "Atomist API for software".